services/av_worker.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file av_worker.js
+ * @author Capuccino
+ * @license BSD-3-Clause
+ */
+global.Promise = require("bluebird");
+const micro = require("micro");
+const https = require("https");
+const {json, send} = micro;
+const config = require("./av.config");
+
+const server = micro(async req => {
+    const data = json(req);
+
+    request("POST", `https://www.virustotal.com/vtapi/v2/url/scan?apikey=${config.apiKey}`, {}, {data}).then(res => {
```

Comment:
Switch to async/await from a Promise, otherwise the async context is somewhat useless.

Reply:
Also I believe `apikey` goes into the body, and I'm not sure exactly what's getting sent since it doesn't say anywhere here what this worker should be getting.